src/logger.rs

```diff
@@ -9,17 +9,16 @@ use chrono::Utc;
 use std::fs;
 #[cfg(not(target_os = "windows"))]
 use std::os::unix::fs::symlink;
-use std::path::Path;
+use std::path::{Path, PathBuf};
 
 pub(crate) struct FilesystemLogger {
-	file_path: String,
+	log_dir: String,
 	level: Level,
 }
 
 impl FilesystemLogger {
 	pub(crate) fn new(log_dir: String, level: Level) -> Result<Self, ()> {
-		let log_file_name =
-			format!("ldk_node_{}.log", chrono::offset::Local::now().format("%Y_%m_%d"));
+		let log_file_name = FilesystemLogger::make_log_file_name();
```

Comment:
why does a log file have to be created here given the changes to how the log function works?